tests/lib/rules/key-spacing.js

```diff
@@ -161,7 +160,7 @@ ruleTester.run("key-spacing", rule, {
             mode: "minimum"
         }]
     }, {
-        code: "obj = { key     :      longName };",
+        code: "obj = { key     :longName };",
```

Comment:
Yeah, that's what threw me off on the issue. I figured given the test there must've been something I was missing.